packages/oui-tile/_variables.less

```diff
@@ -0,0 +1,18 @@
+/* Tile */
+@oui-tile-background-color: #fff;
+@oui-tile-border-color: #dadce0;
+@oui-tile-border-radius: 3px;
```

Comment:
you should use this variable instead: `@oui-global-base-radius` (in `packages/oui/_settings.less`).